### PR TITLE
Information on Organizations and Projects

### PR DESCRIPTION
**Pull request by manojVivek**

### CODE REVIEW

File: docs/organizations-and-projects.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+## Organizations
+Organizations are the top-level umbrella for all the projects in a company. A user can create multiple organizations to separate projects under different companies or teams.
+An organization can have multiple users who have access to all projects under the organization. Users can invite other users to join and manage the organization. Check out the [Invite Users](/docs/invite-users) page for more information on inviting users to an organization.
+
+The name of the organization can typically be the name of the company or the department.
+
+Example: **My Company**.
+
+## Projects
+Projects are the workspaces for profile data within an organization, mainly to collect profiles that belong to a specific application or environment. Profiles within a project can be compared and analysed against each other as required.
```
Comment (brancz) [inline docs/organizations-and-projects.mdx:17]: Maybe add something like "Projects are the unit of separation, data from two projects cannot be queried across each other."

File: docs/organizations-and-projects.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Organizations and Projects
+
+## Organizations
+Organizations are the top-level umbrella for all the projects in a company. A user can create multiple organizations to separate projects under different companies or teams.
```
Comment (brancz) [inline docs/organizations-and-projects.mdx]: I think 

> Organizations are the top-level umbrella for all the projects in a company.

is opinion. We recommend this, but it doesn't have to be that. I think I would start with the general description, something like: "Organizations are collections of projects and where permissions for users are granted. We recommend to create an organization per company."

File: docs/organizations-and-projects.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+The name of the organization can typically be the name of the company or the department.
+
+Example: **My Company**.
+
+## Projects
+Projects are the workspaces for profile data within an organization, mainly to collect profiles that belong to a specific application or environment. Profiles within a project can be compared and analysed against each other as required.
+There can be multiple projects within an organization, and the users of an organization have access to all projects within the organization.
+
+Example: **Production Environment**.
```
Comment (brancz) [inline docs/organizations-and-projects.mdx]: Similar to my comment on how to use organizations, we should make a recommendation on how to use projects. I think something like: "We recommend starting with creating one project and writing all data to a single project, that allows for the best visibility across the entire infrastructure."

File: docs/organizations-and-projects.mdx (hunk 1 above)
Comment (manojVivek) [reply docs/organizations-and-projects.mdx:17]: done, added this.

File: docs/organizations-and-projects.mdx (hunk 2 above)
Comment (manojVivek) [reply docs/organizations-and-projects.mdx]: Fixed this.

File: docs/organizations-and-projects.mdx (hunk 3 above)
Comment (manojVivek) [reply docs/organizations-and-projects.mdx]: done, added this.

File: docs/organizations-and-projects.mdx [hunk 4]
```diff
@@ -1,7 +1,8 @@
 # Organizations and Projects
 
 ## Organizations
-Organizations are the top-level umbrella for all the projects in a company. A user can create multiple organizations to separate projects under different companies or teams.
+Organizations are collections of projects and where permissions for users are granted. A user may create multiple organizations to separate projects under different companies or teams. *We recommend to create an organization per company*.
```
Comment (brancz) [inline docs/organizations-and-projects.mdx]: docusaurus has this cool "Tips" feature, you can see it being used here: https://github.com/parca-dev/docs/blob/main/docs/kubernetes.mdx#kubernetes

I think making this a "Tip" makes sense here. (and for the other one as well)

Comment (manojVivek) [reply docs/organizations-and-projects.mdx]: TIL!

Here is how it looks now:
Screenshot 2023-01-03 at 5 20 27 PM

**Comment by brancz** [comment]
Very nice!!

**Pull request merged**